include/solvers/vof_subequations_interface.h

```diff
@@ -141,6 +160,18 @@ class VOFSubequationsInterface
     return this->subequations_dof_handler[subequation_id];
   }
 
+  /**
+   * @brief Get a pointer to the DoFHandler of the VOF auxiliary physics.
+   *
+   * @return Pointer to the DoFHandler of the VOF auxiliary physics associated
+   * to the set Filtered VOF solution field.
+   */
+  DoFHandler<dim> *
+  get_vof_dof_handler()
+  {
+    return this->dof_handler_vof;
+  }
```

Comment:
Could you not return a reference instead of a pointer? We try to avoid carrying pointer around. The ideal thing would be a const reference :)

Reply:
I based it on `MultiphysicsInterface`, but you are absolutely right! Changes have been made! :)
Maybe eventually the same should be done for `MultiphysicsInterface`; should I open an issue?

Reply:
Yeah I would open an issue :)